src/authenticationinapp/authenticationinapp.h

```diff
@@ -86,7 +86,11 @@ class AuthenticationInApp final : public QObject {
   // Sign In/Up.
   Q_INVOKABLE void signIn();
   Q_INVOKABLE void signUp();
-  Q_INVOKABLE const QString& emailAddress() const;
+  Q_INVOKABLE const QString emailAddress() const;
+
+  Q_INVOKABLE int getVerificationCodeLength() {
```

Comment:
expose it as a property:

`Q_PROPERTY(int verificationCodeLength READ verificationCodeLength CONSTANT)`

Reply:
Makes sense, thanks!